computer_vision/cnn_classification.py

```diff
@@ -0,0 +1,50 @@
+# Convolutional Neural Network
```

Comment:
Module level docstring is better in triple quotes :)
```python
"""
description, references, etc 
"""
```

Reply:
Description added to doctoring